### DISCUSSION TITLE
🏆 GitHub Achievements 🏆

### DISCUSSION THREAD

**Discussion by ThisIs-Developer** [community/community#114421] [Discover: GitHub Best Practices]
# Complete list of all GitHub Profile badges and Achievements

#### Displaying achievements on your profile is completely optional; by default, they can be seen by anyone viewing your public profile.

You can opt out from having achievements displayed on your profile by going to your [profile settings](https://github.com/settings).

<br>

| Badge | Name | How to get                                                                                                                                                       |
| :---: | --- |------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| ![Achievement badge Heart On Your Sleeve](https://github.githubassets.com/images/modules/profile/achievements/heart-on-your-sleeve-default.png) | **Heart On Your Sleeve** | React to something on GitHub with a ❤️ emoji **(Being tested)** |
| ![Achievement badge Open Sourcerer](https://github.githubassets.com/images/modules/profile/achievements/open-sourcerer-default.png) | **Open Sourcerer** | User had PRs merged in multiple public repositories **(Being tested)** |
| ![Achievement badge Starstruck](https://github.githubassets.com/images/modules/profile/achievements/starstruck-default.png) | **Starstruck** | Created a repository that has **16 stars** or [more](#badge-tiers).                                                                                              |
| ![Achievement badge Quickdraw](https://github.githubassets.com/images/modules/profile/achievements/quickdraw-default.png) | **Quickdraw** | Closed an issue or a pull request within 5 min of opening.                                                                                                       |
| ![Achievement badge Pair Extraordinaire](https://github.githubassets.com/images/modules/profile/achievements/pair-extraordinaire-default.png) | **Pair Extraordinaire** | Coauthored in **one** or [more](#badge-tiers) merged pull requests.                                                                                             |
| ![Achievement badge Pull Shark](https://github.githubassets.com/images/modules/profile/achievements/pull-shark-default.png) | **Pull Shark** | **2 pull requests** merged (or [more](#badge-tiers)).                                                                                                            |
| ![Achievement badge Galaxy Brain](https://github.githubassets.com/images/modules/profile/achievements/galaxy-brain-default.png) | **Galaxy Brain** | 2 accepted answers or [more](#badge-tiers).                                                                                                                      |
| ![Achievement badge YOLO](https://github.githubassets.com/images/modules/profile/achievements/yolo-default.png) | **YOLO** | Merged **at least one** pull request without code review .                                                                                                       |
| ![Achievement badge Public Sponsor](https://github.githubassets.com/images/modules/profile/achievements/public-sponsor-default.png) | **Public Sponsor** | Sponsoring open source work via [GitHub Sponsors](https://github.com/sponsors).                                                                                  |
| ![Achievement badge Mars 2020 Contributor](https://github.githubassets.com/images/modules/profile/achievements/mars-2020-contributor-default.png) | **Mars 2020 Contributor** | Contributed code to repositories used in the [Mars 2020 Helicopter Mission](https://github.com/readme/featured/nasa-ingenuity-helicopter). *Now unable to earn.* |
| ![Achievement badge 2020 GitHub Archive Program](https://github.githubassets.com/images/modules/profile/achievements/arctic-code-vault-contributor-default.png) | **Arctic Code Vault Contributor** | Contributed code to a repository in the [2020 GitHub Archive Program](https://archiveprogram.github.com/). *Now unable to earn.*                                 |

<br>

## Badge Tiers

Some Achievements not only have the base version, but also tiers.

| Achievement | Default | Bronze | Silver | Gold |
| --- | :---: | :---: | :---: | :---: |
| **Starstruck** | ![Achievement badge Starstruck](https://github.githubassets.com/images/modules/profile/achievements/starstruck-default.png) | ![Bronze badge Starstruck](https://github.githubassets.com/images/modules/profile/achievements/starstruck-bronze.png) | ![Silver badge Starstruck](https://github.githubassets.com/images/modules/profile/achievements/starstruck-silver.png) | ![Gold badge Starstruck](https://github.githubassets.com/images/modules/profile/achievements/starstruck-gold.png) |
| | 16 stars | 128 stars | 512 stars | 4096 stars <br>[@torvalds](https://github.com/torvalds?achievement=starstruck&tab=achievements) |
| **Pair Extraordinaire** | ![Achievement badge Pair Extraordinaire][pe-default] | ![Bronze badge Pair Extraordinaire][pe-bronze] | ![Silver badge Pair Extraordinaire][pe-silver] | ![Gold badge Pair Extraordinaire][pe-gold] |
| | 1 pull requests <br>[@gomzyakov](https://github.com/gomzyakov?achievement=pair-extraordinaire&tab=achievements) | 10 pull requests | 24 pull requests  | 48 pull requests <br>[@Rongronggg9](https://github.com/Rongronggg9?achievement=pair-extraordinaire&tab=achievements) |
| **Pull Shark** | ![Achievement badge Pull Shark][ps-default] | ![Bronze badge Pull Shark][ps-bronze] | ![Silver badge Pull Shark][ps-silver] | ![Gold badge Pull Shark][ps-gold] |
| | 2 pull requests | 16 pull requests | 128 pull requests | 1024 pull requests <br>[@ljharb](https://github.com/ljharb?achievement=pull-shark&tab=achievements) |
| **Galaxy Brain** | ![Achievement badge Galaxy Brain][gb-default] | ![Bronze badge Galaxy Brain][gb-bronze] | ![Silver badge Galaxy Brain][gb-silver] | ![Gold badge Galaxy Brain][gb-gold] |
| | 2 answers | 8 answers | 16 answers | 32 answers <br>[@ljharb](https://github.com/ljharb?achievement=galaxy-brain&tab=achievements) |
| **Heart On Your Sleeve** | ![Achievement badge Heart On Your Sleeve](https://github.githubassets.com/images/modules/profile/achievements/heart-on-your-sleeve-default.png) | ![Bronze badge Heart On Your Sleeve](https://github.githubassets.com/images/modules/profile/achievements/heart-on-your-sleeve-bronze.png) | ![Silver badge Heart On Your Sleeve](https://github.githubassets.com/images/modules/profile/achievements/heart-on-your-sleeve-silver.png) | ![Gold badge Heart On Your Sleeve](https://github.githubassets.com/images/modules/profile/achievements/heart-on-your-sleeve-gold.png) |
| | ??? | ??? | ??? | ??? |
| **Open Sourcerer** | ![Achievement badge Open Sourcerer](https://github.githubassets.com/images/modules/profile/achievements/open-sourcerer-default.png) | ![Bronze badge Open Sourcerer](https://github.githubassets.com/images/modules/profile/achievements/open-sourcerer-bronze.png) | ![Silver badge Open Sourcerer](https://github.githubassets.com/images/modules/profile/achievements/open-sourcerer-silver.png) | ![Gold badge Open Sourcerer](https://github.githubassets.com/images/modules/profile/achievements/open-sourcerer-gold.png) |
| | ??? | ??? | ??? | ??? |

[ss-bronze]: https://github.githubassets.com/images/modules/profile/achievements/starstruck-bronze.png
[ss-silver]: https://github.githubassets.com/images/modules/profile/achievements/starstruck-silver.png
[ss-gold]: https://github.githubassets.com/images/modules/profile/achievements/starstruck-gold.png

[pe-default]: https://github.githubassets.com/images/modules/profile/achievements/pair-extraordinaire-default.png
[pe-bronze]: https://github.githubassets.com/images/modules/profile/achievements/pair-extraordinaire-bronze.png
[pe-silver]: https://github.githubassets.com/images/modules/profile/achievements/pair-extraordinaire-silver.png
[pe-gold]: https://github.githubassets.com/images/modules/profile/achievements/pair-extraordinaire-gold.png

[ps-default]: https://github.githubassets.com/images/modules/profile/achievements/pull-shark-default.png
[ps-bronze]: https://github.githubassets.com/images/modules/profile/achievements/pull-shark-bronze.png
[ps-silver]: https://github.githubassets.com/images/modules/profile/achievements/pull-shark-silver.png
[ps-gold]: https://github.githubassets.com/images/modules/profile/achievements/pull-shark-gold.png

[gb-default]: https://github.githubassets.com/images/modules/profile/achievements/galaxy-brain-default.png
[gb-bronze]: https://github.githubassets.com/images/modules/profile/achievements/galaxy-brain-bronze.png
[gb-silver]: https://github.githubassets.com/images/modules/profile/achievements/galaxy-brain-silver.png
[gb-gold]: https://github.githubassets.com/images/modules/profile/achievements/galaxy-brain-gold.png

<br>

## Achievement Skin Tone

Some achievements' appearance depends on your Emoji Skin Tone Preference.

You can change your preferred Skin Tone by going to [appearance settings](https://github.com/settings/appearance).

<br>

| **Badge** | 👋 | 👋🏻 | 👋🏼 | 👋🏽 | 👋🏾 | 👋🏿 |
| --- | :---: | :---: | :---: | :---: | :---: | :---: |
| **Starstruck** | ![Default skin tone of Starstruck](https://github.githubassets.com/images/modules/profile/achievements/starstruck-default.png) | ![Light skin tone of Starstruck](https://github.githubassets.com/images/modules/profile/achievements/starstruck-default--light.png) | ![Light-medium skin tone of Starstruck](https://github.githubassets.com/images/modules/profile/achievements/starstruck-default--light-medium.png) | ![Medium skin tone of Starstruck](https://github.githubassets.com/images/modules/profile/achievements/starstruck-default--medium.png) | ![Medium-dark skin tone of Starstruck](https://github.githubassets.com/images/modules/profile/achievements/starstruck-default--medium-dark.png) | ![Dark skin tone of Starstruck](https://github.githubassets.com/images/modules/profile/achievements/starstruck-default--dark.png) |
| **Quickdraw** | ![Default skin tone of Quickdraw][q-default] | ![Light skin tone of Quickdraw][q-light] | ![Light-medium skin tone of Quickdraw][q-light-medium] | ![Medium skin tone of Quickdraw][q-medium] | ![Medium-dark skin tone of Quickdraw][q-medium-dark] | ![Dark skin tone of Quickdraw][q-dark] |

[s-light]: https://github.githubassets.com/images/modules/profile/achievements/starstruck-default--light.png
[s-light-medium]: https://github.githubassets.com/images/modules/profile/achievements/starstruck-default--light-medium.png
[s-medium]: https://github.githubassets.com/images/modules/profile/achievements/starstruck-default--medium.png
[s-medium-dark]: https://github.githubassets.com/images/modules/profile/achievements/starstruck-default--medium-dark.png
[s-dark]: https://github.githubassets.com/images/modules/profile/achievements/starstruck-default--dark.png

[q-default]: https://github.githubassets.com/images/modules/profile/achievements/quickdraw-default.png
[q-light]: https://github.githubassets.com/images/modules/profile/achievements/quickdraw-default--light.png
[q-light-medium]: https://github.githubassets.com/images/modules/profile/achievements/quickdraw-default--light-medium.png
[q-medium]: https://github.githubassets.com/images/modules/profile/achievements/quickdraw-default--medium.png
[q-medium-dark]: https://github.githubassets.com/images/modules/profile/achievements/quickdraw-default--medium-dark.png
[q-dark]: https://github.githubassets.com/images/modules/profile/achievements/quickdraw-default--dark.png

<br>

## Highlights Badges

| Badge | Name | Is it possible to get? | How to achieve |
| --- | --- | --- | --- |
| ![White badge GitHub Pro](https://user-images.githubusercontent.com/65187002/173065531-57dbf8b1-7eb7-4d46-81bf-f2d18c7c9112.svg#gh-dark-mode-only)![Black badge GitHub Pro](https://user-images.githubusercontent.com/65187002/173065669-d1fdb5a7-8895-43cc-8dea-72a511a37e86.svg#gh-light-mode-only) | **Pro** | Yes | Use [GitHub Pro](https://docs.github.com/en/get-started/learning-about-github/githubs-products#github-pro) |
| ![Dark badge Developer Program Member](https://user-images.githubusercontent.com/65187002/173079579-3c393d22-7a13-4e7d-87b8-341fb613d52b.svg#gh-dark-mode-only)![Light badge Developer Program Member](https://user-images.githubusercontent.com/65187002/173079614-33f43a97-1cc2-4228-85e3-ef43836e17c2.svg#gh-light-mode-only) | **Developer Program Member** | Yes | Be a registered member of the [GitHub Developer Program](https://docs.github.com/en/developers/overview/github-developer-program) |
| ![security-bug-bounty-hunter-dark](https://user-images.githubusercontent.com/65187002/173081624-93e3cf1f-50b7-45a4-82b7-1954f66368b9.svg#gh-dark-mode-only)![security-bug-bounty-hunter-light](https://user-images.githubusercontent.com/65187002/173081657-e500d72c-9247-44c2-a3d3-2deff30e1ae7.svg#gh-light-mode-only) | **Security Bug Bounty Hunter** | Yes | Helped out hunting down security vulnerabilities at [GitHub Security](https://bounty.github.com/) |
| ![Light badge GitHub Campus Expert][gce-dark]![Dark badge GitHub Campus Expert][gce-light] | **GitHub Campus Expert** | No | Participate in the [GitHub Campus Program](https://education.github.com/experts) (will open [in August 2024](https://education.github.com/campus_experts)) |
| ![Dark badge Security advisory credit][SAC-dark]![Light badge Security advisory credit][SAC-light] | **Security advisory credit** | Yes | Have your security advisory submitted to the [GitHub Advisory Database](https://github.com/advisories) accepted |
| ![Dark badge Github Stars][stars-dark]![Light badge Github Stars][stars-light] | **GitHub Star** | Yes | Become a [GitHub Star](https://stars.github.com) |
| ![Dark badge Discussion answered](https://user-images.githubusercontent.com/65187002/173078083-15a75f15-b040-4a92-8d70-561a206d9fd9.svg#gh-dark-mode-only)![Light badge Discussion answered](https://user-images.githubusercontent.com/65187002/173078106-28bea542-4620-46ee-837d-defda3e44ca6.svg#gh-light-mode-only) | **Discussion answered** | No | Have  your reply to a discussion marked as the answer |

[gce-dark]: https://user-images.githubusercontent.com/65187002/173082819-b3625c23-bfd6-4492-b828-56ed91c45f52.svg#gh-dark-mode-only
[gce-light]: https://user-images.githubusercontent.com/65187002/173082836-08be81fe-13b7-4acf-9096-e5241d76f237.svg#gh-light-mode-only
[SAC-dark]: https://user-images.githubusercontent.com/65187002/173084051-79a0a626-1c1a-4d60-afdf-50ad001d7b21.svg#gh-dark-mode-only
[SAC-light]: https://user-images.githubusercontent.com/65187002/173084071-5f321da2-b2a9-490b-a524-1b21fa384d7e.svg#gh-light-mode-only
[stars-dark]: images/stars-dark.svg#gh-dark-mode-only
[stars-light]: images/stars-light.svg#gh-light-mode-only

**Comment by queenofcorgis** [comment]
Hi @ThisIs-Developer, 

This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

The Community Discussions Team

Please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).
